packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -276,15 +269,32 @@ export class MetamaskPage implements WalletPage<WalletTypes.EOA> {
     });
   }
 
-  async changeWalletAccountByName(accountName: string) {
+  async changeWalletAccountByName(accountName: string, isClosePage = true) {
     await test.step('Change wallet account', async () => {
       await this.navigate();
       await this.header.accountMenuButton.click();
       await this.accountMenu.clickToAccount(accountName);
-      const accountNumber =
-        this.header.accountMenuButton.getByText(accountName);
+      const accountNumber = this.header.accountMenuButton
+        .locator('..')
+        .getByText(accountName);
       await accountNumber.waitFor({ state: 'visible', timeout: 2000 });
       await this.page.waitForTimeout(2000);
+      if (isClosePage) await this.page.close();
+    });
+  }
+
+  async sendEthTo(address: string, amount: string) {
+    await test.step('Send ETH', async () => {
```

Comment:
Could we change name of step to `Send ${anmount} ETH to ${address}` ? or without addres, but with amount?